versioning/BRM/shmkeys.cpp

```diff
@@ -63,7 +64,7 @@ ShmKeys::ShmKeys()
 string ShmKeys::keyToName(unsigned key)
 {
     ostringstream oss;
-    oss << "InfiniDB-shm-";
+    oss << "MCS-shm-";
```

Comment:
This is an important change that potentially affects external scripts that watch/manipulate shmem files.